packages/lib/src/utils/FileSystem.ts

```diff
@@ -1,14 +1,14 @@
-import * as fs from 'fs';
-import * as path from 'path';
```

Comment:
This may cause an incompatibility with the Ts port. No action needed tho, we'll see...